assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -177,7 +177,8 @@ function MediaPane(props) {
    */
   const insertMediaElement = (resource) => {
     const width = Math.min(resource.width * DEFAULT_DPR, DEFAULT_ELEMENT_WIDTH);
-    return insertElement(resource.type, { resource, width });
+    const isPlaying = resource.type === 'video';
+    return insertElement(resource.type, { resource, width, isPlaying });
```

Comment:
We don't want image elements to have an `isPlaying` property, do we?

How did you deal with avoiding this property to be saved to the db?

Also, doesn't pressing play/pause now add an entry to the history because this changes the element in the story provider? That doesn't really make sense either. This should somehow be meta-data for the item in question, but not part of history, not part of stored data. It's basically something completely new.

Reply:
I was too optimistic and forgot about the history stack.
Solving it!

Reply:
I was too optimistic and forgot about the history stack.
Solving it!